### PR TITLE
Fix: div by 0 when number of Elements is 1

### PR DESCRIPTION
**Pull request by jeremyfelder**
## Describe the changes

This PR fixes a div by 0 edge case in NTT when the number of elements is 1

### CODE REVIEW

**Review by ImmanuelSegol — DISMISSED** [review]
Would it be possible to write a test for this ?

**Review by vhnatyk — DISMISSED** [review]

File: icicle/appUtils/ntt/ntt.cuh [hunk 1]
```diff
@@ -328,7 +328,7 @@ void ntt_inplace_batch_template(
 
     if (is_coset) batch_vector_mult(coset, d_inout, n, batch_size, stream);
 
-    num_threads = min(n / 2, MAX_NUM_THREADS);
+    num_threads = max(min(n / 2, MAX_NUM_THREADS),1);
```
[the remark this answers is not in the document]
Comment (jeremyfelder) [reply icicle/appUtils/ntt/ntt.cuh]: Good catch, fixed [fe2442b](https://github.com/ingonyama-zk/icicle/pull/230/commits/fe2442be3f949bed674fe68891b10046a1c3e9a6)

**Review by ImmanuelSegol — APPROVED** [review]
